src/templates/toolkit/account/api_tokenws.html.tt

```diff
@@ -19,6 +19,18 @@
                     </div>
                 </div>
 
+                <div class="grd-grid-12 grd-parent grd-row-padding">
+                    <div class="grd-grid-4 form_label">
+                        <label for="txtName">[% l('Choose Scopes:') %]</label>
+                    </div>
+                    <div class="grd-grid-8">
+                        <input id="scopes" type="checkbox" name="scopes[]" value="read" checked="checked" /> Read
+                        <input type="checkbox" name="scopes[]" value="trade" /> Trade
+                        <input type="checkbox" name="scopes[]" value="payments" /> Payments
+                        <input type="checkbox" name="scopes[]" value="admin" /> Admin
+                    </div>
+                </div>
```

Comment:
My remark about the loginid-token relationship was about the following. A user logs in once. Later he can switch loginids. So, not knowing about how our system works, he can get the impression that there is one account (namely the one he has logged in to) with just a number of loginids assigned to it. It's not clear when he creates a token that this token includes only one loginid instead of being related to the whole account which includes all his loginids.